File: app/src/main/java/com/riders/thelab/data/remote/dto/directions/Steps.java
package com.riders.thelab.data.remote.dto.directions;

import com.google.gson.annotations.SerializedName;


public class Steps {

    @SerializedName(value = "travel_mode")
    String travelMode;

    @SerializedName(value = "start_location")
    Coordinates startLocation;

    @SerializedName(value = "end_location")
    Coordinates endLocation;
}

File: core/data/src/main/java/com/riders/thelab/core/data/remote/dto/directions/Steps.kt
package com.riders.thelab.core.data.remote.dto.directions

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Steps(
    @SerialName(value = "travel_mode")
    var travelMode: String? = null,

    @SerialName(value = "start_location")
    var startLocation: Coordinates? = null,

    @SerialName(value = "end_location")
    var endLocation: Coordinates? = null
) : java.io.Serializable